index.js

```diff
@@ -489,32 +495,37 @@ class K8sExecutor extends Executor {
                 return this.breaker.runCommand(statusOptions);
             })
             .then(resp => {
-                if (resp.statusCode !== 200) {
-                    throw new Error(`Failed to get pod status:${JSON.stringify(resp.body, null, 2)}`);
-                }
-
-                const status = resp.body.status.phase.toLowerCase();
-
-                if (status === 'failed' || status === 'unknown') {
-                    throw new Error(`Failed to create pod. Pod status is:${JSON.stringify(resp.body.status, null, 2)}`);
-                }
-
+                let error = null;
                 const updateConfig = {
                     apiUri: this.ecosystem.api,
                     buildId,
                     token
                 };
 
-                if (resp.body.spec && resp.body.spec.nodeName) {
-                    updateConfig.stats = {
-                        hostname: resp.body.spec.nodeName,
-                        imagePullStartTime: new Date().toISOString()
-                    };
+                if (resp.statusCode !== 200) {
+                    error = `Failed to get pod status:${JSON.stringify(resp.body, null, 2)}`;
+                    updateConfig.status = STATUS_FAILED;
```

Comment:
should the status be one of these?
https://github.com/screwdriver-cd/data-schema/blob/master/models/build.js#L101-L113

Reply:
yes thnx